src/main/java/com/unb/tracker/UserRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.unb.tracker;
+
+
+import org.springframework.data.repository.CrudRepository;
+// This will be AUTO IMPLEMENTED by Spring into a Bean called userRepository
+// CRUD refers Create, Read, Update, Delete
+
+public interface UserRepository extends CrudRepository<User, Long> {
+
```

Comment:
I would also like to point out something I learned from here: https://spring.io/guides/gs/accessing-data-jpa/

if you want to be able to query for an object based on some attribute, you can just add that method to the repository class and it will somehow magically figure out the mapping for that so when you query based on that object, it'll be built for you. Not sure how combined queries work out though.